src/map/status.cpp

```diff
@@ -12554,6 +12560,10 @@ int32 status_change_start(struct block_list* src, struct block_list* bl,enum sc_
 			tick_time = status_get_sc_interval(type);
 			val4 = tick - tick_time; // Remaining time
 			break;
+		case SC_OVERCOMING_CRISIS:
+			val2 = 3 * val1;
+			val3 = 15000 * val1;
```

Comment:
The status increases the hpmax but each cast should also heal the player (no number displayed on screen) of.. val3 (probably)

Reply:
So, using status_heal is fine in this case?